inc/cron_pull.inc

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Pull cron function for Loco Translate module.
+ */
+
+/**
+ * Pull assets & translations from Loco.
+ *
+ * @param string $langcode
+ *   The langcode to pull from.
+ * @param array $settings
+ *   The configurations needed to pull from Loco.
+ */
+function loco_translate_cron_pull($langcode, array $settings) {
+  $loco_pull = \Drupal::service('loco_translate.loco_api.pull');
+  $translations_import = \Drupal::service('loco_translate.translations.import');
+  $file_system = \Drupal::service('file_system');
+
+  $loco_pull->setApiClientFromConfig();
+  $status = $settings['status'] ?? NULL;
+  $response = $loco_pull->fromLocoToDrupal($langcode, $status);
+
+  /** @var \Drupal\file\FileInterface $file */
+  $file = file_save_data($response->__toString(), 'translations://');
```

Comment:
this will create a new file entry in the database. Is this required for the `TranslationImport` and underlying services ?

I propose:
```php
$tmp_file = tmpfile();
file_put_contents((string) $response, $tmp_file);
$report = $translations_import->fromFile($tmp_file, $langcode);
// ...
unlink($tmp_file);
```

Reply:
I would prefer to use - at least - the `file_save_data`  to use in a near futur the `filesystem->saveData`